etc/crontab.xml

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
-    <group id="yotpo">
+    <group id="yotpo_core_orders_sync">
```

Comment:
This will improve the cron performance. If we separate the cron jobs into different groups , each group will be transformed in to unique PHP processes and they will execute independently. 

A cron group is a logical group that enables you to easily run cron for more than one process at a time

https://devdocs.magento.com/guides/v2.4/config-guide/cron/custom-cron-ref.html

Reply:
Thanks.